src/aind_codeocean_api/codeocean.py

```diff
@@ -318,6 +323,8 @@ def register_result_as_data_asset(
         asset_description: Optional[str] = "",
         mount: Optional[str] = None,
         tags: Optional[List] = None,
+        custom_metadata: Optional[dict] = None,
+
```

Comment:
I think this extra space should get cleaned up by running black?

Reply:
done